docs-source/content/userguide/managing-fmw-domains/soa-suite/Monitor-Exporter-setup.md

```diff
@@ -0,0 +1,286 @@
+# Weblogic Monitor Exporter Setup to monitor the SOA domain instance
+
+You can monitor the SOA instance using Prometheus and Grapahna by exporting the metrics from the domain instance using the weblogic monitor exporter to the prometheus.
+
+This document provides the steps to setup the weblogic monitor exporter to push the data to prometheus in order to monitor the SOA instance.
+
+## Build Weblogic Monitor Exporter  
+
+Create a working directory and make sure it has full access persmissions. 
```

Comment:
what does "full access permissions" mean?  The example commands below do not show changing permissions at all.  Permissions is spelled wrong.